client/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["es2015", "stage-0", "react"]
```

Comment:
Why do we have `.babelrc`'s (namely the same `.babelrc`) in all the directories?